xdsl/printer.py

```diff
@@ -1,12 +1,12 @@
 from __future__ import annotations
 
+from dataclasses import dataclass, field
+from enum import Enum
 from frozenlist import FrozenList
-
```

Comment:
The changes here are basically ordering imports (which I find absolutely okay). I want to point out though that we still have an open [issue](https://github.com/xdslproject/xdsl/issues/147) about that, that I feel could use a decision.

Reply:
Discussion has opened for this. I guess a whole of a fix on the order can be done in another autonomous PR.